Examples/Idealgas.py

```diff
@@ -0,0 +1,131 @@
+from openmmtools.testsystems import IdealGas
+from openmm import unit
+
+
+# Use the IdealGas example from openmmtools to initialize particle positions and topology
+# For this example, the topology provides the masses for the particles
+
+n_particles = 216
+temperature = 298 * unit.kelvin
+pressure = 1 * unit.atmosphere
+mass = unit.Quantity(39.9, unit.gram / unit.mole)
```

Comment:
Yes. Good catch. 

I'm wondering if it would be good to have examples be self contained (i.e., not require openmmtools).  I think the openmmtools dependence is definitely good for tests/unit tests.   It would definitely not be hard to have some simple routines for initializing particles.  I don't think we need to do that at this time.